test/display.js

```diff
@@ -0,0 +1,34 @@
+import { setup } from "./_helpers.js";
+import { describe, expect, test } from "vitest";
+import * as displayRules from '../src/_rules/display.js';
```

Comment:
Isnt this exposed from _rules index? would be cleaner to import from there instead, if possible 💡 

```suggestion
import * as displayRules from '../src/_rules/display.js';
```

Reply:
Yep, this should be able to be `#rules` for the import - [defined here](https://github.com/warp-ds/drive/blob/master/package.json#L14)

Reply:
Maybe we can fix those imports in a better matter in a separate PR? Another example is here https://github.com/warp-ds/drive/blob/master/test/static.js#L3